### DISCUSSION TITLE
Start clifm with max name length unset?

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#208] [Q&A] [answered]
Most of the time when I start clifm I instantly press `ctrl+alt+l` to toggle `toggle-max-name-len` and see full file names. Is it possible to start with this option toggled and at the same time not lose the possibility to toggle it again (to again limit file names to 20 chars)?

I know I can change `MaxFilenameLen` setting to eg 100, but then I can't toggle filenames back to 20 with the key shortcut.

**Comment by leo-arch** [comment, marked as answer]
Hi @tkossak. Let me take a look at it.

**Reply by leo-arch** [reply]
Done! Set `TrimNames` to `false` in the config file or run with `--no-trim-names` to disable file names trimming (you can still toggle this option via <kbd>Ctrl-Alt-l</kbd> using the value set in `MaxFilenameLen` as usual).

Please let me know if it works as intended.

**Reply by tkossak** [reply]
Works perfectly, thank you very much! You're doing awesome job with this wonderful tool.
